docs/spec.md

```diff
@@ -1614,9 +1614,9 @@ the following IEEE-754 operations:
 * `LE`: `compareQuietLessEqual`.
 * `LT`: `compareQuietLess`.
 
-For floating-point element types and `compare_type = TOTALORDER`, the op
+For floating-point element types with `compare_type = TOTALORDER`, the op
```

Comment:
If we are updating "and" to "with" here, then we'll want to do the same for the preceding paragraph for consistency.